### ISSUE TITLE
Unlisting dev versions below 4.0

### ISSUE THREAD

**Issue by HavenDV** [unoplatform/uno#9358]
## What would you like to be added:
Have you thought about unlisting dev versions below 4.0? This will still allow them to be downloaded, but will greatly speed up the IDE when working with Uno NuGet packages. Right now, opening this list in the IDE takes about 20 seconds.
![image](https://user-images.githubusercontent.com/3002068/180827652-a00d7513-dd82-43f5-9f9a-524b72c49da4.png)

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

**Comment by Arlodotexe** [comment]
Frankly, Visual Studio should be able to handle this. Rider doesn't seem to have any issue with it, and even https://nuget.info/ can handle scrolling through them all without issue.

**Comment by jeromelaban** [comment]
@Arlodotexe indeed. VS does not virtualize that list, but still, we have in the plans to unlist intermediate versions of packages to make this part easier :)

**Comment by jeromelaban** [comment]
This is being slowly worked on as part of our release process, to unlist many packages. Uno.WinUI and Uno.UI have been processed, at the rythm of API rate limiting :)

**Comment by jeromelaban** [comment]
This is done. We're keeping the dev bits listed for one version prior to the current stable.
